controllers/substitute.js

```diff
@@ -114,14 +114,14 @@ var Compendium = require('../lib/model/compendium');
  */
 function createFolder(passon) {
     return new Promise((fulfill, reject) => {
-        var outputPath = path.join(config.fs.compendium, passon.id) + '/data';
+        var outputPath = path.join(config.fs.compendium, passon.id, '/data');
```

Comment:
Good choice to use `path.join(..)`, but then you can leave out the leading `/` before `data`. [Here and several other places.]